src/recon.rs

```diff
@@ -1,6 +1,25 @@
 use crate::src::msac::dav1d_msac_decode_bool_equi;
 use crate::src::msac::MsacContext;
 
+/// This is a macro that defines a function
```

Comment:
If you are describing remaining work, I'd recommend starting the comment (and similar comments) with `TODO: ` such that we can easily grep through the code to find outstanding work. This is also what we're doing on line 9.

Reply:
Good idea, I'll add that.

Reply:
Added the `TODO` in cb05c1d68cf17c2a26597425757cb80d66be5177 along with a longer comment explaining how we might remove the macro.